data/src/main/java/me/androidbox/data/alarm_manager/AlarmReceiver.kt

```diff
@@ -1,17 +1,24 @@
 package me.androidbox.data.alarm_manager
 
 import android.app.NotificationManager
+import android.app.PendingIntent
 import android.content.BroadcastReceiver
 import android.content.Context
 import android.content.Intent
+import android.net.Uri
 import androidx.core.app.NotificationCompat
+import androidx.core.app.TaskStackBuilder
+import androidx.core.net.toUri
 import me.androidbox.data.R
 import me.androidbox.data.alarm_manager.AlarmSchedulerImp.Companion.EXTRA_DESCRIPTION
 import me.androidbox.data.alarm_manager.AlarmSchedulerImp.Companion.EXTRA_ID
 import me.androidbox.data.alarm_manager.AlarmSchedulerImp.Companion.EXTRA_TITLE
 
 class AlarmReceiver : BroadcastReceiver() {
 
+    companion object {
+        private const val EVENT_DEEPLINK = "https://androidbox.me/event?agendaId="
```

Comment:
I would rename agendaId to **id**

Since agendaId doesnt make much sense

Reply:
Also, you could have this on a Constants class
Since you will have to add it to the navgraph also

Reply:
@MKiperszmid Adds these to a constants class